hackathon_site/dashboard/frontend/src/components/dashboard/TeamCard/TeamCard.tsx

```diff
@@ -2,27 +2,28 @@ import React from "react";
 import styles from "./TeamCard.module.scss";
 import Container from "@material-ui/core/Container";
 import Typography from "@material-ui/core/Typography";
-import Button from "@material-ui/core/Button";
 import TitledPaper from "components/general/TitledPaper/TitledPaper";
 
-const TeamCard = ({ members, teamCode, handleEditTeam }) => {
-    const title = teamCode === "" ? "Team" : "Team " + teamCode;
+interface TeamProps {
+    members?: string[];
+    teamCode?: string;
+}
 
+const TeamCard = ({ members, teamCode }: TeamProps) => {
+    const title = teamCode === "" ? "Team" : "Team " + teamCode;
+    // waiting for the team store
+    if (!members) {
+        return null;
+    }
     return (
         <TitledPaper title={title}>
-            {members.map((member, i) => (
+            {members?.map((member, i) => (
                 <Container className={styles.name} key={i}>
                     <Typography variant="body2" noWrap>
                         {member}
                     </Typography>
                 </Container>
             ))}
-
```

Comment:
we don't need to delete this

Reply:
should we keep the handleEditTeam as a prop?

Reply:
Yupp let's keep it